guides/assets/stylesrc/_main.scss

```diff
@@ -792,6 +792,50 @@ body.guide {
             color: $rf-brand !important;
             text-decoration: underline !important;
           }
+
+          a.active {
+            position: relative;
```

Comment:
please arrange them alphabetically

Reply:
i copied the same style used for sidebar navigation section and changed some values.

https://github.com/rails/rails/blob/e43991b3ea26186633b74c1ca03396926d07c613/guides/assets/stylesrc/_main.scss#L1013